app/controllers/files_controller.rb

```diff
@@ -30,6 +30,10 @@ def preserved
     response.headers['Content-Disposition'] = "attachment; filename=#{filename}"
     response.headers['Last-Modified'] = Time.now.utc.rfc2822 # HTTP requires GMT date/time
     self.response_body = file_content
+  rescue Preservation::Client::NotFoundError => e
+    response.headers['Last-Modified'] = Time.now.utc.rfc2822 # HTTP requires GMT date/time
```

Comment:
Why are you setting this header?

Reply:
b/c it was set above?  should it be left empty?

Reply:
I don't know why it's useful for a 404.

Reply:
i'll remove it.

Reply:
removed.